src/3.0/Dockerfile

```diff
@@ -82,25 +83,27 @@ RUN wget https://github.com/INCF/MUSIC/archive/master.tar.gz && \
 RUN git clone https://github.com/nest/nest-simulator.git && \
   cd nest-simulator && \
   git checkout master && \
+  python3 -m pip install -r ./doc/requirements.txt
```

Comment:
Link to `./doc/requirements.txt` changed. It is now:` ./extras/nest-simulator-doc-requirements.txt `. So the line should be:

`python3 -m pip install -r ./extras/nest-simulator-doc-requirements.txt && \`